src/Symfony/Component/Console/Resources/completion.fish

```diff
@@ -0,0 +1,29 @@
+# This file is part of the Symfony package.
+#
+# (c) Fabien Potencier <fabien@symfony.com>
+#
+# For the full copyright and license information, please view
+# https://symfony.com/doc/current/contributing/code/license.html
+
+function _sf_{{ COMMAND_NAME }}
+    set sf_cmd (string split ' ' (commandline))
+    set c (math (count $sf_cmd) - 1)
```

Comment:
Looking at this code, it seems like we always only autocomplete values at the end of the command line input.

I see almost the same code in https://github.com/fish-shell/fish-shell/issues/6922 , except that they are talking about using `commandline --tokenize --current-process --cut-at-cursor` and `commandline --current-token`. to get the actual index of the cursor.

Do we need to add something similar?

Reply:
Adding `-c` seems to fix it.

I made a quick video capture, but I don't find how to insert it. So here's a small picture.

![image](https://user-images.githubusercontent.com/917449/145803316-16212e43-738f-4444-a26b-68e8ec1c3e15.png)

Reply:
Thanks for the quick response!

I didn't test it yet, but I think we have to call `commandline` twice. At least, in the bash implementation I take the input after the cursor into account when completing. And `-c` means "cut at cursor", removing the input after it (right?).

Also, I think it's probably better to use the `commandline --tokenize` buildin instead of splitting on a whitespace (not sure if there is a difference, but at least it's probably closer to what the shell really does).

I.e:
```
    set sf_cmd (commandline -o)
    set c (math (count (commandline -c)) - 1)
```

Reply:
Thanks for the feedback. I updated the code.